RNA-Seq-de-novo/docker/README.md

```diff
@@ -0,0 +1,29 @@
+[![Docker Automated buil](https://img.shields.io/docker/automated/bgruening/galaxy-training-exome-seq.svg?maxAge=2592000)](https://hub.docker.com/r/bgruening/galaxy-training-exome-seq/)
+[![Docker Pulls](https://img.shields.io/docker/pulls/bgruening/galaxy-training-exome-seq.svg?maxAge=2592000)](https://hub.docker.com/r/bgruening/galaxy-training-exome-seq/)
+[![Docker Stars](https://img.shields.io/docker/stars/bgruening/galaxy-training-exome-seq.svg?maxAge=2592000)](https://hub.docker.com/r/bgruening/galaxy-training-exome-seq/)
+
+Galaxy Docker repository for Exome sequencing training material :whale: :eyes:
+====
+
+:whale: This Docker Image contains everything what is needed to run this training, including an interactive Galaxy Tour.
+
+# Requirements
+
+- [Docker](https://docs.docker.com/installation/) for Linux / Windows / OSX
```

Comment:
Depending on the tools and the training data ... so hard to tell. In general no. Every special requirement should be stated in the readme file imho.